index.js

```diff
@@ -38,8 +38,22 @@ alfy.alfred = {
 
 alfy.input = process.argv[2];
 
-alfy.output = arr => {
-	console.log(JSON.stringify({items: arr}, null, '\t'));
+const isDefined = x => x !== null && x !== undefined;
+
+const formatItemArg = item => {
+	if (isDefined(item) && isDefined(item.variables)) {
```

Comment:
This has to be expanded. It's also possible to provide variables for mods like this

```js
alfy.output([
    {
        title: 'Unicorn',
        arg: '🦄',
        variables: {
            color: 'white'
        },
        mods: {
            alt: {
                title: 'Rainbow',
                arg: '🌈',
                variables: {
                    color: 'red'
                }
            }
        }
    }
]);
```

If you press the <kbd>alt</kbd> key, it will now show the `Rainbow` instead.